docs/rules/jsx-uses-react.md

```diff
@@ -1,6 +1,8 @@
-# Disallow React to be incorrectly marked as unused (react/jsx-uses-react)
+# Disallow React to be incorrectly marked as unused (`react/jsx-uses-react`)
 
-💼 This rule is enabled in the following [configs](https://github.com/jsx-eslint/eslint-plugin-react#shareable-configurations): `all`, `recommended`. This rule is disabled in the following configs: `jsx-runtime`.
```

Comment:
There's one feature I need to implement to indicate which configs disable a rule. I'll update the PR in the coming days after implementing that feature tracked below.

https://github.com/bmish/eslint-doc-generator/issues/21

Reply:
This has been implemented in: https://github.com/bmish/eslint-doc-generator/releases/tag/v0.14.0